src/cards/thermostat-card/utils.ts

```diff
@@ -16,10 +17,22 @@ export const getStepSize = (hass: HomeAssistant, entity: ClimateEntity): number
     return entity.attributes.target_temp_step ?? systemStep;
 };
 
+export const supportTargetTemperatureRange = (entity: ClimateEntity) => supportsFeature(entity, 2);
```

Comment:
I'm running into an issue here where my thermostat supports setting target ranges for auto/heat_cool but does not expose them for just heat or cool, so this is coming back as `true` but I've got undefined for both `target_temp_high` and `target_temp_low`. I'm thinking this should be guarded by both the feature check and the mode check to make sure we are only taking advantage of this feature in the modes it is supposed to be used for?